# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1384776396 -3600
#      Mon Nov 18 13:06:36 2013 +0100
# Node ID 92b3343e28c499fd88b191605934f896079cb97b
# Parent  a4bc3ad4c703fedccd492fca0a7545d2b4f49dca
* main-window.cc (set_window_layout): remove double setVisible ()

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -910,17 +910,16 @@ main_window::set_window_layout (QSetting
   show ();  // main window is ready and can be shown (as first window)
 
   // show floating widgets after main win to ensure "Octave" in central menu
   foreach (octave_dock_widget *widget, float_and_visible)
     {
 #if not defined (Q_OS_WIN32)
       widget->make_window ();
 #endif
-      widget->setVisible (true);
       if (settings->value ("DockWidgets/" + widget->objectName () + "_minimized").toBool ())
         widget->showMinimized ();
       else
         widget->setVisible (true);
     }
 
 }
 
